project2/index.html

```diff
@@ -0,0 +1,106 @@
+                    <li>1 pound fresh or frozen raw shrimp</li>
+                    <li>2 tablespoons extra-virgin olive oil</li>
+                    <li>1 shallot</li>
+                    <li>5 cloves garlic</li>
+                    <li>1/2-1 teaspoon crushed red pepper flakes</li>
+                    <li>1 teaspoon kosher salt</li>
+                    <li>1 teaspoon black pepper</li>
+                    <li>24 ounces mixed vegetables</li>
+                    <li>1/4 cup dry white wine</li>
+                    <li>Zest and juice of 1 large or 2 small lemons</li>
+                    <li>3 tablespoons freshly grated Parmesan cheese</li>
+                    <li>2 tablespoons chopped fresh parsley</li>
+
+                </ul>
+            </p>
+        </section>
+
+        <section id="fas fa-utensil-spoon">
+            <h2><b>Instructions</b></h2>
+
+            <ol>
+                <li>Bring a large pot of salted water to a boil and cook the pasta until al dente, according to
+                    package directions. Reserve 1/4 cup pasta water, drain remaining water, and then toss
+                    with a bit of olive oil to prevent sticking. Set aside.</li><br>
+
+                <li>Meanwhile, rinse the shrimp and pat dry. Heat 1 tablespoon olive oil in a large skillet over
+                    medium-high heat. Add the shallot and cook until fragrant, about 2 minutes. Add the garlic
+                    and cook 1 additional minute. Add the shrimp, and then sprinkle with half the amount of
+                    red pepper (1/4 or 1/2 teaspoon depending upon your desired spice level), 1/2 teaspoon
+                    salt, and 1/2 teaspoon black pepper. Sauté, stirring to coat with garlic, just until cooked
+                    through (both sides will be pink and opaque), about 3 to 4 minutes. Remove to a plate and
+                    set aside.</li><br>
+
+                <li>Heat the remaining 1 tablespoon olive oil over medium high. Add the vegetables and the
+                    remaining 1/4 or 1/2 teaspoon red pepper, 1/2 teaspoon salt, and 1/2 teaspoon black
+                    pepper. Cook until the vegetables are hot and lightly sautéed, 4 to 5 minutes. Add the
+                    white wine and return the shrimp to the skillet. Cook for 2 minutes, allowing the wine to
+                    reduce. Stir in the lemon zest and juice and remove from heat.</li><br>
+
+                <li>Add the pasta to the skillet and toss. If the pasta is too dry, add a bit of the reserved pasta
+                    water. Serve warm, topped with Parmesan and parsley.</li>
+            </ol>
+        </section>
+
+        <section id="fas fa-clock">
+            <ul>
+                <li><i class="fas fa-clock"> PREP TIME: 15 MINS</i></li>
+                <li> <i class="fas fa-clock"> COOK TIME: 15 MINS</i></li>
+            </ul>
+        </section>
+
+        <i class="fab fa-twitter"></i>
```

Comment:
1. Consider putting your social icons in an unordered list `ul`
1. These icons would also need to be wrapped in an `a` tag that links to the various social networks, for example:
```
<a href="https://twitter.com"><i class="fab fa-twitter"></i></a>
```